pkg/virt-controller/watch/vmi.go

```diff
@@ -424,7 +424,7 @@ func isPodReady(pod *k8sv1.Pod) bool {
 				return false
 			}
 		} else {
-			if containerStatus.Ready == false {
+			if containerStatus.Name != "istio-proxy" && containerStatus.Ready == false {
```

Comment:
As an uninitiated newbie, two questions:

1. where is this name - ```istio-proxy``` - defined? Is it guaranteed to always have this exact form?
2. why do we think it's safe to proceed even if the Istio proxy container is not ready yet?

Both questions should probably be clarified with corresponding comments in the code itself.

Reply:
Done

Reply:
@SchSeba thanks. Anything about the name of the container? Is it guaranteed to be this way in all cases?

Reply:
This is the default if you really want you can change this in the istio-injection-sidecar-configmap

Reply:
I guess ideally we would accommodate for a different name if configmap is modified, but perhaps it's not critical.